File: Task3/Go/inout.go
package main

import (
	"fmt"
	"io/ioutil"
	"os"
)

func main() {
	inputFile := "input.txt"
	outputFile := "output.txt"

	content, err := ioutil.ReadFile(inputFile)
	if err != nil {
		fmt.Println("Error reading file:", err)
		return
	}

	err = ioutil.WriteFile(outputFile, content, 0644)
	if err != nil {
		fmt.Println("Error writing file:", err)
	}
}

File: Task3/Elixr/inout.exs
defmodule FileCopy do
  def run do
    # Define the file paths
    input_file = "input.txt"
    output_file = "output.txt"

    # Read the content from the input file
    case File.read(input_file) do
      {:ok, content} ->
        # Write the content to the output file
        case File.write(output_file, content) do
          :ok -> IO.puts("File successfully copied to #{output_file}")
          {:error, reason} -> IO.puts("Error writing to file: #{reason}")
        end

      {:error, reason} ->
        IO.puts("Error reading file: #{reason}")
    end
  end
end
FileCopy.run()

